src/engraving/rw/read410/tread.cpp

```diff
@@ -3383,9 +3383,28 @@ void TRead::read(Pedal* p, XmlReader& e, ReadContext& ctx)
     if (p->score()->mscVersion() < 301) {
         ctx.addSpanner(e.intAttribute("id", -1), p);
     }
+
+    if (p->score()->mscVersion() < 420) {
+        p->setBeginText(String());
+        p->setContinueText(String());
+        p->setEndText(String());
+    }
+
     while (e.readNextStartElement()) {
         const AsciiStringView tag(e.name());
         if (readStyledProperty(p, tag, e, ctx)) {
+        } else if (tag == "lineVisible") {
+            if (TRead::readProperty(p, tag, e, ctx, Pid::LINE_VISIBLE)) {
```

Comment:
It looks like this should either be 
```suggestion
        } else if (TRead::readProperty(p, tag, e, ctx, Pid::LINE_VISIBLE)) {
```
or 
```suggestion
        } else if (tag == "lineVisible") {
            TRead::readProperty(p, e, ctx, Pid::LINE_VISIBLE));
```